frontend/packages/core/src/Input/stories/text-field.stories.tsx

```diff
@@ -8,13 +8,22 @@ import TextField from "../text-field";
 export default {
   title: "Core/Input/TextField",
   component: TextField,
+  argTypes: {
```

Comment:
So to make this easier, we can auto import the available options for storybook. It's a bit hacky for some of them but the gist is this:

https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/Input/text-field.tsx#L385
```diff
- const TextField = React.forwardRef<HTMLDivElement, TextFieldProps>(TextFieldRef);
+ export const TextField = React.forwardRef<HTMLDivElement, TextFieldProps>(TextFieldRef);
```

https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/Input/stories/text-field.stories.tsx#L6
```diff
- import TextField from "../text-field";
+ import { TextField } from "../text-field";
```

There's a weird bug that doesn't allow default exports to work correctly and they must be a named export